src/Worker.Extensions.DurableTask/Execution/DurableFunctionExecutor.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Functions.Worker.Invocation;
+using Microsoft.DurableTask;
+using Microsoft.DurableTask.Worker;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.DurableTask.Execution;
+
+internal partial class DurableFunctionExecutor(
+    IFunctionExecutor inner,
+    ExtendedSessionsCache extendedSessionsCache,
+    IDurableTaskFactory factory,
+    IOptions<DurableTaskWorkerOptions> options,
+    IExceptionPropertiesProvider? exceptionPropertiesProvider)
```

Comment:
Not sure if this should be optional or not. It seems like the intent was for it to be optional?

```suggestion
    IExceptionPropertiesProvider? exceptionPropertiesProvider = null)
```

Reply:
You are correct, this should be optional :)